cli/src/lib/plugin/webpack.config.js

```diff
@@ -0,0 +1,255 @@
+// Based on CRA Webpack config
```

Comment:
See https://github.com/facebook/create-react-app/blob/v4.0.3/packages/react-scripts/config/webpack.config.js

The Webpack config from react-scripts v4.0.3 was made for Webpack 4, but this PR uses Webpack 5, and so could not be used directly.

Reply:
The main thing missing from this config is support for service workers. This is how react-scripts does it: https://github.com/facebook/create-react-app/blob/v4.0.3/packages/react-scripts/config/webpack.config.js#L712-L724